### PR TITLE
Rtrieu/docs 7510 applicationlaunch view android

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

Adds the same ApplicationLaunch view details that are in the iOS page to the Android page, per user feedback described in [DOCS-7510](https://datadoghq.atlassian.net/browse/DOCS-7510)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7510]: https://datadoghq.atlassian.net/browse/DOCS-7510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rtrieu** [comment]
@plousada Hi Pedro, would you mind reviewing this PR to make sure it makes sense? Thank you!

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ The following diagram illustrates the RUM event hierarchy:
 
 {{< img src="real_user_monitoring/data_collected/event-hierarchy.png" alt="RUM Event hierarchy" style="width:50%;" >}}
 
+## Application launch
+
+During initialization, the RUM Android SDK creates a view called "ApplicationLaunch". This view's start time matches the start of the Android process, and can be used to track your application launch time.
+
+The ApplicationLaunch view includes any logs, actions, and resources created before your first call to `startView`. Use the duration of this view to determine time to first view. This view has an action, `application_start`, with a duration equal to the amount of time from process start until the call to `applicationDidBecomeActive`.
```
Comment (plousada) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md:43]: `applicationDidBecomeActive` is an iOS callback and won't exist on Android, so we should remove it. 

We can state that the `application_start` duration will be equal to the two bullet points below, depending on Android version.

**Review by plousada — CHANGES_REQUESTED** [review]

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md]: Hi @plousada, I've updated it. What do you think?

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ The following diagram illustrates the RUM event hierarchy:
...
 {{< img src="real_user_monitoring/data_collected/event-hierarchy.png" alt="RUM Event hierarchy" style="width:50%;" >}}
 
+## Application launch
+
+During initialization, the RUM Android SDK creates a view called "ApplicationLaunch". This view's start time matches the start of the Android process, and can be used to track your application launch time.
+
+The `ApplicationLaunch` view includes any logs, actions, and resources created before your first call to `startView`. Use the duration of this view to determine time to first view. This view has an action, `application_start`, with a duration that depends on Android version:
+
+- In *Android 7.0* and above, this view/action captures the period before any application code is executed (right before `Application.onCreate`) and when the first RUM event is recorded.
+- In versions before * Android 7.0*, the view/action captures the period between the `RumFeature` class loads and when the first RUM event is recorded.
```
Comment (drichards-87) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md:46]:
```suggestion
- In versions before *Android 7.0*, the view/action captures the period between the `RumFeature` class loads and when the first RUM event is recorded.
```

**Review by drichards-87 — APPROVED** [review]
Left one small suggestion and approved the PR.

**Pull request merged**